fast_and_reliable.py

```diff
@@ -0,0 +1,30 @@
+import time
+
+
+def average_runtime(date_structure):
+    """
+    The function checks 1000 times how many times the word zwitterion appears in the data structure.
+    :param date_structure: List or set of words.
+    :return: Run time of the function.
+    """
+    start = time.time()
+    counter_word = 0
+    loop_number = 0
+
+    for word in date_structure:
```

Comment:
Instead we can just run the loop 1000 times and inside it to check if "zwitterion" is in  data_structure. I think it will be more simple and short.